test/tracing/reactnativenavigation.test.ts

```diff
@@ -72,19 +78,15 @@ describe('React Native Navigation Instrumentation', () => {
         contexts: expect.objectContaining({
           trace: expect.objectContaining({
             data: {
-              route: {
-                name: 'Test',
-                componentName: 'Test',
-                componentId: '0',
-                componentType: 'Component',
-                hasBeenSeen: false,
-                passProps: {},
-              },
-              previousRoute: null,
+              [SEMANTIC_ATTRIBUTE_ROUTE_NAME]: 'Test',
+              [SEMANTIC_ATTRIBUTE_ROUTE_COMPONENT_ID]: '0',
+              [SEMANTIC_ATTRIBUTE_ROUTE_COMPONENT_TYPE]: 'Component',
+              [SEMANTIC_ATTRIBUTE_ROUTE_HAS_BEEN_SEEN]: false,
```

Comment:
Q: there's no componentName?

Reply:
I removed it, as both route name and component name was component name in the implementation and so it seems redundant.